configure.ac

```diff
@@ -295,6 +295,9 @@ fi
 
 LIBS="$LIBS $GLIB_LIBS"
 
+GLIB_CFLAGS=" $GLIB_CFLAGS"
+GLIB_CFLAGS="${GLIB_CFLAGS// -I/ -isystem}"
```

Comment:
wot

Reply:
.... I don't think I intended to commit that.... If you're curious, it is a useful feature, if you want to compile your own code with -pedantic or more warnings, since -isystem will hide warnings in those directories ( as opposed to -I )